metadata_annotation_farland.ipynb

```diff
@@ -0,0 +1,2164 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    # Am I the only one? If so, need to improve performace... although I already tried many many different ways ...cry
</pre><p>That's weird. I'll try it myself...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>